lib/grammers-client/src/types/media.rs

```diff
@@ -63,6 +69,37 @@ impl Photo {
             P::Photo(photo) => photo.id,
         }
     }
+
+    /// Get photo thumbs.
```

Comment:
This could use a bit more information. What does "thumbs" stand for? Can I somehow rely on the order of the returned photos? If I have a photo media, can this be empty? What can I use the thumbnails for when I already have the photo? Can I use all the thumbnails to download a specific size, or some won't work?

Reply:
I've added a bit more docs